psd-web/app/controllers/investigations/coronavirus_related_controller.rb

```
@@ -0,0 +1,20 @@
+module Investigations
+  class CoronavirusRelatedController < ApplicationController
+    def show
+      @investigation = Investigation.find_by(pretty_id: params.require(:investigation_pretty_id)).decorate
+    end
+
+    def update
+      investigation = Investigation.find_by(pretty_id: params.require(:investigation_pretty_id))
+
+      investigation.assign_attributes(params.require(:investigation).permit(:coronavirus_related))
+      create_audit = investigation.coronavirus_related_changed?
+
+      investigation.save!
```

Decided to let this blow up. There is no validation on this field so the chances of anything going wrong are pretty low.

---

There is validation here: https://github.com/UKGovernmentBEIS/beis-opss-psd/blob/master/psd-web/app/models/investigation.rb#L15

It is possible for this to be a value other than `true` or `false` although not through ordinary usage.

---

I guess it's also possible that we might somehow not submit an answer at all (somehow the radios aren't pre-selected, or the selection fails).